src/less/modules/report/form.less

```diff
@@ -0,0 +1,150 @@
+.form {
+  margin: auto;
+  padding: 80px 0 30px;
+  max-width: 1135px;
+  width: 90%;
+  font-size: 18px;
+
+  &__column {
+    &_20 {
+      width: 20%;
+      float: left;
+      margin-top: 10px;
+    }
+
+    &_25 {
+      width: 25%;
+      float: left;
+      margin-top: 10px;
+    }
+
+    &_40 {
+      width: 40%;
+      float: left;
+      margin-top: 10px;
+    }
+
+    &_75 {
+      width: 75%;
+      float: left;
+      margin-top: 10px;
```

Comment:
You don't need to repeat `float: left; margin-top: 10px;` for every of `&_20`, `&_25`, `&_40`, `&_75`. You can put it one time in the following place:
```
&__column {
  float: left;
  margin-top: 10px;
```